pkg/serviceaccount/jwt.go

```diff
@@ -139,7 +139,7 @@ type Validator interface {
 	NewPrivateClaims() interface{}
 }
 
-func (j *jwtTokenAuthenticator) AuthenticateToken(tokenData string) (user.Info, bool, error) {
+func (j *jwtTokenAuthenticator) AuthenticateToken(auds []string, tokenData string) (user.Info, bool, error) {
```

Comment:
I expected this to make use of auds if provided, prior to `j.validator.Validate`

```
if len(auds) > 0 && len(public.Audience) > 0 && !contains(auds, public.Audience) {
  return nil, false, 
}
```

Reply:
Done.